spanner_orm/condition.py

```diff
@@ -225,13 +225,67 @@ def _validate(self, model):
     del model
 
 
+class OrCondition(Condition):
+  """Used to join multiple conditions with an OR in a Spanner query."""
+
+  def __init__(self, *condition_lists):
+    super().__init__()
+    if len(condition_lists) < 2:
+      raise error.SpannerError(
+          'OrCondition requires at least two lists of conditions')
+    self.condition_lists = condition_lists
+    self.all_conditions = []
+    for conditions in condition_lists:
+      self.all_conditions.extend(conditions)
```

Comment:
nit: I prefer 
```
self.all_conditions += conditions
```

Reply:
i believe += is less efficient than extend